### ISSUE TITLE
Disabled wireless device throws OSError for `sensor_temperatures`

### ISSUE THREAD

**Issue by dcousens** [giampaolo/psutil#1104]
> systemctl stop wpa_supplicant@wlp4s0.service

then

``` python
Python 3.6.1 (default, Mar 27 2017, 00:27:06)
[GCC 6.3.1 20170306] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import psutil
>>> psutil.sensors_temperatures()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/site-packages/psutil/__init__.py", line 2212, in sensors_temperatures
    rawdict = _psplatform.sensors_temperatures()
  File "/usr/lib/python3.6/site-packages/psutil/_pslinux.py", line 1116, in sensors_temperatures
    current = float(cat(base + '_input')) / 1000.0
  File "/usr/lib/python3.6/site-packages/psutil/_pslinux.py", line 299, in cat
    return f.read().strip()
OSError: [Errno 5] Input/output error
```

fixed by

> systemctl start wpa_supplicant@wlp4s0.service

``` python
>>> psutil.sensors_temperatures()
{'acpitz': [shwtemp(label='', current=39.0, ... etc
```

uname
> Linux arch 4.11.6-3-ARCH

**Comment by dcousens** [comment]
Maybe related to https://github.com/giampaolo/psutil/issues/1009,  but not fixed by https://github.com/giampaolo/psutil/commit/3f67acceb089029563c423517b5508bdc9cb6356 (obviously)

**Comment by dcousens** [comment]
tl;dr

``` 
cat /sys/class/hwmon/hwmon4/name
> iwlwifi
cat /sys/class/hwmon/hwmon4/temp1_input
40000
```

> systemctl stop wpa_supplicant@wlp4s0.service
equals

```
cat /sys/class/hwmon/hwmon4/temp1_input
> cat: /sys/class/hwmon/hwmon4/temp1_input: Input/output error
```

**Comment by dcousens** [comment]
Catching (and swallowing) the `IOError`,  and `continue` to continue appending other sensors IMHO is the suitable solution

**Comment by giampaolo** [comment]
This should have been fixed by now (https://github.com/giampaolo/psutil/issues/1129)

**Comment by dcousens** [comment]
Thanks @giampaolo
